charts/quickwit/templates/metastore-statefulset.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: apps/v1
+kind: StatefulSet
```

Comment:
Unlike the indexers and searchers, the metastore instances are stateless, so I don't think using `StatefulSet` is necessary.

Reply:
Ok, I'll change it to a deployment